documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -109,10 +109,11 @@ Let's see the options for the `index_schema_management_strategy` property:
 [options="header"]
 |===============
 |Value|Definition
-|NONE|Indexes will not be created nor deleted.
-|MERGE|Missing indexes and mappings will be created, mappings will be updated if there are no conflicts.
-|CREATE|Indexed will be deleted if existing and then created. This will delete all content from the index!
-|CREATE_DELETE|Similarly to 'CREATE' but will also delete the index at shutdown. Commonly used for tests.
+|`NONE`|Indexes will not be created nor deleted.
+|`MERGE`|Missing indexes and mappings will be created, mappings will be updated if there are no conflicts.
+|`CREATE`|**The default**: existing indexes will not be altered, missing indexes will be created.
+|`RECREATE`|Indexed will be deleted if existing and then created. This will delete all content from the index!
+|`RECREATE_DELETE`|Similarly to `RECREATE` but will also delete the index at shutdown. Commonly used for tests.
```

Comment:
I think Hibernate's wording is prior to the JPA wording, and they're not consistent.
If I had to stick to an existing wording, I'd rather choose JPA's, which is a standard and is much more explicit ("create", "drop-and-create", etc.).

I can switch to the JPA wording if you wish, but I fear that the one we have now will harm some careless users, which is why I'd really like those terms to change before the release.